### DISCUSSION TITLE
Change calendar module display options

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15911] [How to] [answered]
Hi,

I have two queries regarding the calendar module. 

1. By default, how to set the calendar module to display all days (all 7 days).?
2. It either displays for processing activities or future activities. ? How to display both simultaneously.? 
![calendar](https://user-images.githubusercontent.com/96602176/162563399-33375362-b898-40ee-8680-b5ef60efdd60.png)

Thanks in advance.

with regards,
Rajasekaran

**Comment by rskrzypczak** [comment, marked as answer]
1. Unfortunately, you can't change it from the GUI. You have to change it directly in the file.
   Empty the array.
  `config\Modules\Calendar.php`
   public static $HIDDEN_DAYS_IN_CALENDAR_VIEW = [];
2. This cannot be changed from the configuration level, it requires modifying the code.

**Reply by Ilamparithi-Tamizhanangu** [reply]
Thanks for solution. Now i can able to all the dates in the calendar.
